src/main/java/seedu/address/logic/commands/applicant/AddApplicant.java

```diff
@@ -30,12 +30,12 @@
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
-            + PREFIX_NRIC + "S12345678D "
+            + PREFIX_NRIC + "S1234567D "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_DATEAPPLIED + "21-3-2022 "
-            + PREFIX_TAG + "new applicant "
+            + PREFIX_DATEAPPLIED + "2022-3-21 "
```

Comment:
The month of date applied should be 2022-03-21.

Reply:
Ok changed